sql-server-2016-fci-existing-vnet-and-ad/dsc/PrepSFCI.ps1

```diff
@@ -153,4 +153,4 @@ function Get-NetBIOSName
             return $DomainName
         }
     }
-} 
```

Comment:
not sure why the browser editor added this change. I just edited line #84

Reply:
Thanks.  Paths are fixed now.